docker-compose.yaml

```diff
@@ -27,7 +32,7 @@ services:
 
   keto:
     container_name: keto
-    image: oryd/keto:v0.6.0-alpha.1-sqlite
+    image: oryd/keto:v0.12.0
```

Comment:
Maybe, it's better to use the same version as we have on _IaC_ and _ory-client_ - 0.11 ?

Reply:
+1

Reply:
Done